### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ApolloZhu** [UWAppDev/uwappdev.github.io#3]
While our current website can be navigated using the tab key, unfortunately, Voice Over is unable to pick up any information.

**Comment by personalizedrefrigerator** [comment]
Okay! I can work on that. Can you give me more information about Voice Over? Is it specific to an operating system? Is Voice Over able to read the main content of the page?
I have added search functionality to proposed. Depending on how the narration application works, a hidden link that causes this element to focus could be added at the top of the page. Does Voice Over read items that have style="display: none" set? Does tabIndex affect the order of narration? Is there any other information I should be aware of?

**Comment by personalizedrefrigerator** [comment]
## TODO
 - [x] Skip to content link.
 - [ ] High contrast option.
 - [ ] Easier navigability with tab and arrow keys.
 - [x] Test with Ubuntu's screen reader (Firefox).
 - [ ] Test with Ubuntu's screen reader (Chromium).
 - [ ] Test with Windows Narrator (Edge).
 - [ ] Test with Windows Narrator (Chrome).
 - [ ] Test with Windows Narrator (Firefox).
 - [ ] Test with OSX narration program.
 - [ ] Test with iOS narration program.

**Comment by personalizedrefrigerator** [comment]
We use `tabindex > 0` for dropdowns. Per https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard, me may want to consider changing this.
